guests-and-visitors.rst

```diff
@@ -3,12 +3,12 @@ Guests & Visitors
 
 Guests & visitors are **welcome in the Hackspace**.
 
-You should not let people into the space if you don't know them, or are unsure if they are a member. If you do then they become your guest and you are responsible for them. Feel free to challenge people to authenticate to GateKeeper (by swiping their card against the door) if you don't know that they are a member.
+You should not let people into the space if you don't know them, or are unsure if they are a member. If you do then they become your guest and you are responsible for them. Feel free to challenge people to authenticate to GateKeeper (by swiping their card against the door) if you don't know that they are a member. You are under no obligation to answer the door if the doorbell rings.
 
-Included in this are both **children** and **animals** - it is common courtesy to ask on the Google Group before attending the Hackspace with either an animal or a child. The exception to this notification is on Open Days and Evenings - members should assume that animals or children may be present at those times.
+Included in this are both **children** and **animals** - it is common courtesy to ask on the Discord or Google Groups before attending the Hackspace with either an animal or a child. The exception to this notification is on Open Days and Evenings - members should assume that animals or children may be present at those times.
 
 Please do not leave your animal, child or guest unattended in the Hackspace. They are your responsibility and must be attended at all times.
 
-Guests cannot use tools that require an induction, such as the laser cutter, Myford lathe or 3D Printer.
+Guests cannot use tools that require an induction, such as the laser cutter, Myford lathe, Bridgeport milling machine, or table saw. You can check whether a tool requires an induction by searching for the tool on the Wiki.
```

Comment:
Worth adding the Warco lathe, planer thicknesser, resin printer and embroidery machine here for a mostly complete list?